packages/node_modules/pouchdb-find/src/utils.js

```diff
@@ -200,19 +118,42 @@ function uniq(arr) {
   return Array.from(new Set(arr));
 }
 
+/**
+ * Callbackifyable wrapper for async functions
+ *
+ * @template T, Args
+ * @param {(...args: Args) => Promise<T>} fun
+ * @returns {<CBArgs = [...Args, (err: any, value: T) => void], InnerArgs extends Args | CBArgs>(...innerArgs: InnerArgs) => InnerArgs extends CBArgs ? void : Promise<T>}
+ *
+ * @example
+ * const fn = resolveToCallback(async () => { return 42; })
+ * // with callback:
+ * fn((num) => { ... })
+ * // with await:
+ * const num = await fn()
+ */
+export function resolveToCallback(fun) {
```

Comment:
@garethbowen what's your preference for reviews:
- export in signature (when it changes, than it's all in one block) or
- export in object block (two detached changes in diff, but often more obvious that an export change)

Reply:
I slightly prefer putting it in the object block so if I'm calling the module I can easily see all exported functions, but I don't feel too strongly about it.

Reply:
Thx for the feedback.
Just changed the export in https://github.com/pouchdb/pouchdb/pull/8860/commits/f94883c7a8c0ef8ce99e38ff1e7e5be9c32e944b

I'll keep the object style from now on.